### ISSUE TITLE
Error when trying to run tflite model on Arduino Nano 33 BLE sense

### ISSUE THREAD

**Issue by WednesdayICEY** [eloquentarduino/EloquentTinyML#72]
I was trying to run tflite model on Arduino Nano 33 BLE sense, I downloaded the required libraries: EloquentTinyML and tflm_cortexm, but errors were reported when compiling. 
libraries\tflm_cortexm\tflm_cortexm.a(fully_connected.cpp.o): In function `tflite::(anonymous namespace)::Eval(TfLiteContext*, TfLiteNode*)':`
Arduino\libraries\tflm_cortexm\src/tensorflow/lite/kernels/internal/reference/integer_ops/fully_connected.h:164: undefined reference to `tflite::tensor_utils::UnpackDenseInt4IntoInt8(signed char const*, int, signed char*)'`
libraries\tflm_cortexm\tflm_cortexm.a(fully_connected_common.cpp.o): In function `tflite::CalculateOpDataFullyConnected(TfLiteContext*, TfLiteFusedActivation, TfLiteType, TfLiteTensor const*, TfLiteTensor const*, TfLiteTensor const*, TfLiteTensor*, tflite::OpDataFullyConnected*)':`
Arduino\libraries\tflm_cortexm\src\tensorflow\lite\micro\kernels/fully_connected_common.cpp:62: undefined reference to `tflite::GetQuantizedConvolutionMultipler(TfLiteContext*, TfLiteTensor const*, TfLiteTensor const*, TfLiteTensor const*, TfLiteTensor*, double*)'`
Arduino\libraries\tflm_cortexm\src\tensorflow\lite\micro\kernels/fully_connected_common.cpp:78: undefined reference to `tflite::CalculateActivationRangeQuantized(TfLiteContext*, TfLiteFusedActivation, TfLiteTensor*, long*, long*)'`
collect2.exe: error: ld returned 1 exit status
How can I fix it?
My model has FulltConnected and Softmax layers
